app/scripts/ui.js

```diff
@@ -16,28 +12,21 @@ import EthQuery from 'eth-query'
 import StreamProvider from 'web3-stream-provider'
 import log from 'loglevel'
 import launchMetaMaskUi from '../../ui'
-import { setupMultiplex } from './lib/stream-utils'
-import setupSentry from './lib/setupSentry'
 import ExtensionPlatform from './platforms/extension'
+import { setupMultiplex } from './lib/stream-utils'
 import {
   ENVIRONMENT_TYPE_FULLSCREEN,
   ENVIRONMENT_TYPE_POPUP,
 } from './lib/enums'
 import { getEnvironmentType } from './lib/util'
+/* eslint-disable import/first,import/order */
```

Comment:
Is this still needed? I can't tell why it was added

Reply:
removed!